### ISSUE TITLE
Microsoft Teams Beta Modules Missing

### ISSUE THREAD

**Issue by rorywcollins** [PowerShell/PowerShellGallery#119]
Unable to download Microsoft Teams beta modules, as repo appears to be gone:  https://www.poshtestgallery.com/packages/MicrosoftTeams/

**Comment by ThomasNieto** [comment]
You're using the test gallery. Please use https://powershellgallery.com

**Comment by rorywcollins** [comment]
@ThomasNieto - correct; per [MS documentation](https://docs.microsoft.com/en-us/powershell/module/teams/add-teamchanneluser?view=teams-ps) this is where the beta modules reside.  Has the location for the beta modules (e.g. 1.0.25) changed?  I don't see them in the production gallery, but perhaps I'm not looking in the right place...

**Comment by edyoung** [comment]
As I understand it,
https://www.powershellgallery.com/packages/MicrosoftTeams/
<https://www.powershellgallery.com/packages/MicrosoftTeams/1.1.4> has both
the preview version of the teams module (1.1.3-preview) and the production
version 1.1.4.

https://docs.microsoft.com/en-us/MicrosoftTeams/teams-powershell-release-notes
explains
more about the Teams module versioning scheme. (I'll be honest - it
confuses me!)

In general nobody should be distributing or downloading modules (preview or
not) via poshtestgallery. That is for testing *the gallery* not for
distributing test modules.

**Comment by rorywcollins** [comment]
@edyoung - appreciate the clarification.  I was after Add-TeamChannelUser, but in looking at the 1.1.3-preview version, it appears to be there (it's not in 1.1.4).  The documentation link I provided above is still referencing poshtestgallery, as well as several public blog posts such as [this one](https://medium.com/@joaquin.guerrero/adding-bulk-users-to-teams-private-channels-8c9c8e563900), or [this one](https://office365itpros.com/2019/11/19/teams-updates-powershell-module-private-channels/), so it appears to have been distributed there at one time, hence the confusion.

**Comment by Petri-X** [comment]
On the main page of PowerShellTestGallery you have view:
![image](https://user-images.githubusercontent.com/43988151/90363672-bfab0480-e06b-11ea-97f3-1a84a0c03ff1.png)

But as @rorywcollins mentioned, Teams (which is on the list) is not working anymore.

Why not add a test on Teams' PowerShell test page what has happening and where to go next. This:
```
Oops!
Error 404 - Page Not Found
```
Is pretty odd.

**Comment by ThomasNieto** [comment]
@joeyaiello / @SydneyhSmith Can either of you two reach out to MS Teams to update their process to publish preview modules to the production gallery instead of the test gallery? It might also be good to lock down the test gallery to only allow your team to publish modules there.

**Comment by SydneyhSmith** [comment]
Yes absolutely @ThomasNieto thanks for helping respond to this issue, we are currently engaged with them internally.
